src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,30 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
The logger is imported and used in this file, but the task description requires configuring the logger with appenders for both File and Console. Ensure that this configuration is added to meet the task requirements.